roles/wordpress-sync/tasks/backup_db.yml

```diff
@@ -0,0 +1,76 @@
+---
+
+- name: Perform mysqldump
+  include_role:
+    name: sql-command
+    tasks_from: mysql-dump
+  vars:
+    sql_host: "wordpress.{{ target }}.internal"
+    executing_user: "{{ rds_dbs_data.wordpressdb_user}}"
+    executing_user_password: "{{ rds_dbs_data.wordpressdb_pass }}"
+    target_db_name: "{{ rds_dbs.0.db_name|default('wordpress') }}"
+
+- name: Compress our sql file
+  archive:
+    path: "{{ sql_file_path }}"
+    dest: "{{ sql_file_path }}.zip"
+    format: zip
+  become: true
+  delegate_to: "{{ 'master.' + vpc_name +'.internal' }}"
+  run_once: yes
+
+- name: Pull down our archive
+  fetch:
+    src: "{{ sql_file_path }}.zip"
+    dest: "/tmp/."
+  delegate_to: "{{ 'master.' + vpc_name +'.internal' }}"
+  run_once: yes
+
+- name: Unpack our archive
+  unarchive:
+    copy: no
+    src: "/tmp/localhost/{{ sql_file_path }}.zip"
+    dest: "{{ playbook_dir }}/uploads-content/"
+  delegate_to: localhost
+
+- name: Overwrite our old sql file
+  shell: "mv {{ playbook_dir }}/uploads-content/{{ sql_file_name }} {{ playbook_dir }}/uploads-content/wordpress.sql"
+  run_once: true
+  delegate_to: localhost
+
+- name: Replace domain name with placeholder
```

Comment:
I need an `if` on this  with the domain_to_replace being set, won't work for current wordpress.sirius-opg.digital` though that may be moot